src/Symfony/Bundle/FrameworkBundle/Command/ServerStartCommand.php

```diff
@@ -177,25 +182,44 @@ private function isOtherServerProcessRunning($address)
     }
 
     /**
+     * Determine the absolute file path for the router script based on user input and the environment.
```

Comment:
OK, I've updated the comment to be more specific in 4be37573055d64418d9d9246ea14c4ecaa03da77.